grep.js

```diff
@@ -9,8 +9,6 @@ const pickTests = (filename, foundTests, cypressConfig) => {
   // each test name is a list of strings
   // [suite name, suite name, ..., test name]
 
-  console.log('picking tests to run in file %s', filename)
```

Comment:
This was very noisy, it logged a multiple lines with a long full filepath to each file